extensions/lorem-ipsum/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "lorem-ipsum",
+  "title": "Lorem Ipsum",
+  "description": "Generate placeholder content",
+  "icon": "command-icon.png",
+  "author": "AntonNiklasson",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "paragraph",
+      "title": "Generate Paragraph",
+      "subtitle": "",
+      "description": "",
+      "mode": "no-view"
+    },
+    {
+      "name": "sentence",
+      "title": "Generate Sentence",
+      "subtitle": "",
```

Comment:
Same here: You might want to add "Lorem Ipsum" as subtitle here (shown as subtitle for the command in root search).